### PR TITLE
Action Blocks | Testing | Update include path and add tests

### PR DESCRIPTION
**Pull request by danielamorse**
## Jira

http://vjira2:8080/browse/BDS-1273

## Summary

Update the path to the `action-block` subcomponent. Add test to check that including the Twig subcomponent directly renders properly.

Add additional tests to check that deprecated props still work.

## Details

The `action-block.twig` adapter file was not updated when we changed the subcomponent filename from `_action-block.twig` to `_action-block-item.twig`. This update fixes that.

Also, the following props were deprecated and tests have been added to check that each one still works:
- `maxItemsPerRow` to `max_items_per_row`
- `align` to `valign`
- `border` to `borderless`

## How to test

- Review the files changed
- See the two new test snapshots and verify this is the HTML we would expect to see given the tests run

### CODE REVIEW

File: packages/components/bolt-action-blocks/__tests__/action-blocks.js [hunk 1]
```diff
@@ -54,6 +54,23 @@ describe('<bolt-action-blocks> Component', () => {
     expect(results.html).toMatchSnapshot();
   });
 
+  // Subcomponent
+  test('Subcomponent renders as expected', async () => {
+    const results = await render(
+      '@bolt-components-action-blocks/_action-blocks-item.twig',
+      {
```
Comment (remydenton) [inline packages/components/bolt-action-blocks/__tests__/action-blocks.js]: What do you think about updating this test to replace this:
`@bolt-components-action-blocks/_action-blocks-item.twig`
With this:
`@bolt-components-action-blocks/action-block.twig`
?

The later is what's actually happening on the Drupal end and is in theory what we're trying to still support until 3.x.  Should be identical output actually.

Comment (sghoweri) [reply packages/components/bolt-action-blocks/__tests__/action-blocks.js]: @danielamorse @remydenton what's the consensus on this? 

Should we get this updated first before I merge this in?

Comment (remydenton) [reply packages/components/bolt-action-blocks/__tests__/action-blocks.js]: It's not critical-- more a point about testing philosophy than anything.  My review shouldn't block merging if there's urgency.

Comment (danielamorse) [reply packages/components/bolt-action-blocks/__tests__/action-blocks.js]: @remydenton 100%, that is the better test. Updated. cc @sghoweri

**Pull request merged**